src/event-helper.js

```diff
@@ -50,11 +50,12 @@ export function createCustomEvent(win, type, detail, opt_eventInit) {
  * @param {string} eventType
  * @param {function(!Event)} listener
  * @param {boolean=} opt_capture
```

Comment:
Any reason not to make this
`@param {boolean|EventListenerOptions}` opt_catureOrOptions